src/auspex/pulse_calibration.py

```diff
@@ -342,8 +342,7 @@ def phase_estimation( data_in, vardata_in, verbose=False):
         curGuess = possibles[0]
         if verbose == True:
             print('Current Guess: %f'%(curGuess))
```

Comment:
Can we swap out `print` in favor of `logger.info`, or is there concern users might not get the message depending on their debug level?